src/core_plugins/kibana/common/field_formats/types/__tests__/duration.js

```diff
@@ -0,0 +1,41 @@
+import expect from 'expect.js';
+import { DurationFormat } from '../duration';
+
+describe('Duration Format', function () {
+
+  test({ inputFormat: 'seconds', outputFormat: 'humanize' })
+    (-60, 'minus a minute')
```

Comment:
Interesting style, but I'm thinking this would be a little less mind bending as:

```js
defineTests({
  inputFormat: 'seconds',
  outputFormat: 'humanize',
  fixtures: [
    [ 60, 'a minute' ],
    // or 
    { input: -60, output: 'minus a minute' },
  ]
})
```